tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -152,6 +152,7 @@ private void OnBeginRequest(object sender, EventArgs eventArgs)
                 string httpMethod = httpRequest.HttpMethod.ToUpperInvariant();
                 string url = httpContext.Request.GetUrlForSpan(tracer.TracerManager.QueryStringManager);
                 var tags = new WebTags();
+                tags.SetTag("component", "aspnet"); // TODO unsure why this is using WebTags and not AspNetTags
```

Comment:
This integration doesn't add any of the tags there (no route, controller, area).

Rider says:
> uses of `AspNetTags` not found
Maybe we can delete that type?